test/plugins/convertPathData.14.svg

```diff
@@ -17,7 +17,7 @@
     <path d="M15 10a5 5 0 1 1 5-5l-5 5"/>
     <path d="M41.008.102a5.006 5.006 0 0 1 3.849 3.705"/>
     <path d="M7.234 19.474a5 5 0 0 1-4.933-8.683"/>
-    <path d="M60 0a5 5 0 1 0 .001 10.001A5 5 0 0 0 60 0z"/>
+    <path d="M60 0a5 5 0 1 0 .001 10.001 5 5 0 0 0 0-10.001z"/>
```

Comment:
ok right, but I think it's random, I'll try to find a case where it's the opposite, round shorter and toFixed longer